src/test-fbp/converter-error-expected.txt

```diff
@@ -0,0 +1 @@
+WRN: ./src/lib/common/sol-types.c:242 sol_irange_division() Division by zero: 10, 0
```

Comment:
Tweaking the debug output will work for this case, but maybe that's not a one-size-fits-all solution. Instead of having a verbatim `-expected.txt` file, maybe one with a regex or some other globbing pattern would work?

    WRN: .*sol-types.c.* Division by zero: 10,0